cursor_test.go

```diff
@@ -104,23 +104,69 @@ func TestCursorSelectError(t *testing.T) {
 
 	// Try to select a column that doesn't exist.
 	c := r.Select("notfound")
-	
+
 	var hasNext bool
 	for c.Next() {
 		hasNext = true
 	}
-	
+
 	if hasNext {
 		t.Errorf("Next returned true, expected false")
 	}
-	
+
 	err = c.Err()
 	if err == nil {
 		t.Errorf("Expected error")
 	}
 	if err.Error() != "no field with name: notfound" {
 		t.Errorf("Unexpected error: %s", err.Error())
 	}
-	
-	
+
+}
+
+func TestCursorSelectStripe(t *testing.T) {
+	r, err := Open("./examples/demo-11-none.orc")
+	if err != nil {
+		t.Fatalf("Expected no error, got %v", err)
+	}
+
+	expectedStripes := 385
+
+	numStripes, err := r.NumStripes()
```

Comment:
What's the purpose of this check? It seems to duplicate `TestNumStripes`.

Reply:
@deafgoat This is just included to stop the test prematurely if the number of stripes is not as expected. It's not the purpose of the test, but should catch any errors in case the behavior of `r.NumStripes()` or the test file is changed.